### PR TITLE
Adding job for upsert organizations

### PR DESCRIPTION
**Pull request by rodri042**

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@rodri042 Looks good to me, but we should also tell the application to consume proper CSS and JS. Are you planning to do that in another PR?

PS: consuming CSS and JS here is trickier than in laboratory, since they are consumed from angular.

File: lib/classroom/event/upsert_organization.rb [hunk 1]
```diff
@@ -1,9 +1,19 @@
-class Classroom::Event::UpsertOrganization
-  class << self
-    def execute!(organization)
-      Classroom::Database.connect_transient! organization['name'] do
-        Classroom::Collection::Organizations.upsert! organization
+module Office
```
Comment (rodri042) [inline lib/classroom/event/upsert_organization.rb]: `// TODO:` Classsroom

File: lib/tasks/organizations.rake [hunk 2]
```diff
@@ -0,0 +1,5 @@
+namespace :organizations do
+  task listen: :environment do
+    Mumukit::Nuntius::EventConsumer.start 'office'
```
Comment (aguspina) [inline lib/tasks/organizations.rake]: TODO: 'classroom'

File: lib/classroom/event/upsert_organization.rb [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
-    def execute!(organization)
-      Classroom::Database.connect_transient! organization['name'] do
-        Classroom::Collection::Organizations.upsert! organization
+module Classroom
+  module Event
+    class UpsertOrganization
+      def self.execute!(payload)
+        organization = payload['organization']
+
+        Classroom::Database.connect_transient! organization['name'] do
```
Comment (flbulgarelli) [inline lib/classroom/event/upsert_organization.rb]: @aguspina are we replicating the Organizations collection in each tenant?

Comment (aguspina) [inline lib/classroom/event/upsert_organization.rb]: Yup, and it has only one docuent

**Comment by aguspina** [comment]
LGTM

**Pull request merged**